lib/ansible/module_utils/cnos.py

```diff
@@ -0,0 +1,3130 @@
+                        command = command + bgpNeighborArg3
+                    else:
+                        retVal="Error-304"
+                        return retVal
+                elif(bgpNeighborArg2 == "loopback"):
+                    value = checkSanityofVariable(deviceType, "bgp_neighbor_update_loopback", bgpNeighborArg3)
+                    if(value == "ok"):
+                        command = command + bgpNeighborArg3
+                    else:
+                        retVal="Error-305"
+                        return retVal
+                else:
+                    value = checkSanityofVariable(deviceType, "bgp_neighbor_update_vlan", bgpNeighborArg3)
+                    if(value == "ok"):
+                        command = command + bgpNeighborArg3
+                    else:
+                        retVal="Error-306"
+                        return retVal
+            else:
+                command = command + bgpNeighborArg2
+        else:
+            retVal="Error-303"
+            return retVal
+                
+    elif(bgpNeighborArg1 == "weight"):
+        command = command + bgpNeighborArg1 + " "
+        value = checkSanityofVariable(deviceType, "bgp_neighbor_weight", bgpNeighborArg2)
+        if(value == "ok"):
+            command = command + bgpNeighborArg2
+        else:
+            retVal="Error-302"
+            return retVal
+    
+    else:
+        retVal = "Error-301"
+        return retVal
+    
+    command = command +"\n"
+    #debugOutput(command)
+    retVal = retVal + waitForDeviceResponse(command, prompt, timeout, obj)
+    command = "exit \n"
+    retVal = retVal + waitForDeviceResponse(command, "(config-router)#", timeout, obj)  
+    return retVal
+#EOM
+
+def bgpAFConfig(obj, deviceType, prompt, timeout, bgpAFArg1, bgpAFArg2, bgpAFArg3, bgpAFArg4, bgpAFArg5, bgpAFArg6):
+    retVal = ""
+    command = ""
+    timeout = timeout
+    if(bgpAFArg1 == "aggregate-address"):
+        command = command + bgpAFArg1 + " "
+        value = checkSanityofVariable(deviceType, "bgp_aggregate_prefix", bgpAFArg2)
+        if(value == "ok"):
+            command = command + bgpAFArg2 + " "
+            if(bgpAFArg2 == None):
+                command = command
+            elif(bgpAFArg2 == "as-set" or bgpAFArg2 == "summary-only" ):
+                command = command + bgpAFArg2 + " "
+                if(bgpAFArg3 == None):
+                    command = command
+                elif(bgpAFArg2 == "as-set" ):
+                    command = command + "summary-only"
+                else:
+                    command = command + "as-set"
+            else:
+                retVal="Error-297"
+                return retVal
+        else:
+            retVal="Error-296"
+            return retVal
+    elif(bgpAFArg1 == "client-to-client"):
+        command = command + bgpAFArg1 + " reflection " 
+    elif(bgpAFArg1 == "dampening"):
+        command = command + bgpAFArg1 + " "
+        if(bg
```

Comment:
We have fixed this issue that was reported by shippable in cnos.py and have updated the branch.  We have not used shippable yet, but have just enabled our repo to have shippable check it, however, still trying to figure out a simple shippable.yml file to have it scan our Python files.